selfhost/typechecker.jakt

```diff
@@ -463,48 +468,74 @@ struct Typechecker {
         return true
     }
 
-    function add_function_to_scope(mut this, parent_scope_id: ScopeId, name: String, function_id: FunctionId, span: Span) throws -> bool {
+    function add_function_to_scope(mut this, parent_scope_id: ScopeId, name: String, function_ids: [FunctionId], span: Span) throws -> bool {
```

Comment:
is this actually adding an overload set? if so it should probably be called add_overload_set. well, really just the scope-local part of a potential overload set, right?

Reply:
good point. I guess overload_set is a good name, although I then should probably change the store to a set just to make sure the name matches.